src/apps/reservation-list/modal/delete-reservation/delete-reservation-modal.tsx

```diff
@@ -74,16 +74,46 @@ const DeleteReservationModal: FC<DeleteReservationModalProps> = ({
   return (
     <Modal
       modalId={modalId}
-      classNames="modal-cta"
+      classNames="modal-cta modal-padding"
       closeModalAriaLabelText={t("deleteReservationModalCloseModalText")}
       screenReaderModalDescriptionText={t(
         "deleteReservationModalAriaDescriptionText"
       )}
     >
-      <DeleteReservationContent
-        deleteReservation={() => removeSelectedReservations()}
-        reservationsCount={reservations.length}
-      />
+      {["idle", "pending"].includes(requestStatus) && (
+        <DeleteReservationContent
+          deleteReservation={() => removeSelectedReservationsHandler()}
+          reservationsCount={reservations.length}
+          deletionStatus={requestStatus}
+        />
+      )}
+      {requestStatus === "success" && (
+        <ModalMessage
+          title={t("deleteReservationModalSuccessTitleText")}
+          subTitle={t("deleteReservationModalSuccessStatusText")}
```

Comment:
Can we tell the user how many reservations were deleted?

Reply:
Yes ;) I skipped it to keep spent coding time down. But you are right, we need it.